packages/http/src/models/Account.ts

```diff
@@ -22,6 +22,29 @@ function toBalance(value: number | undefined, type: CurrencyType): Balance | und
   return new Balance(value, type)
 }
 
+export interface HTTPStatsObject {
+  last_week: Array<TimelineStats>
+  last_month: Array<TimelineStats>
+  last_day: Array<TimelineStats>
+}
+
+export interface TimelineStats {
+  timestamp: string
+  balance: number
+}
+
+export class AccountStats {
```

Comment:
should this be in a separate file maybe?

Reply:
Because the endpoint is `/accounts/id/stats` I added it under Accounts. So you call `client.accounts.getStats('my-address')` thats why I put it in Account.
I can also move it out to its own object, maybe `client.accountStats.get('my-address')`. Let me know what you think

Reply:
i'm fine with the call being nested under `client.accounts.getStats`. I was just thinking since it's a class, maybe we'd want it in its own file to make searching for it easier, but i don't feel too strongly about it